lib/matplotlib/backends/backend_ps.py

```diff
@@ -299,31 +299,28 @@ def create_hatch(self, hatch):
         name = 'H%d' % len(self._hatches)
         linewidth = rcParams['hatch.linewidth']
         pageheight = self.height * 72
-        self._pswriter.write("""\
+        self._pswriter.write(f"""\
   << /PatternType 1
      /PaintType 2
      /TilingType 2
-     /BBox[0 0 %(sidelen)d %(sidelen)d]
-     /XStep %(sidelen)d
-     /YStep %(sidelen)d
+     /BBox[0 0 {sidelen} {sidelen}]
```

Comment:
Do we want to be defensive and keep the formatting information?

Reply:
that they're numbers (`d`) rather than something else?  if anything passing the wrong *value* seems more likely to happen than passing the wrong type.

Reply:
I didn't see that `sidelen = 72` is hard coded just a few lines before. So the risk is actually only in a later code change.

Reply:
I think it's a small risk, but it's much easier to debug a Python TypeError than a broken postscript file. Postscript and pdf viewers don't really show very good error messages.